### ISSUE TITLE
Official container image

### ISSUE THREAD

**Issue by jcmoraisjr** [corazawaf/coraza-spoa#37]
Hi, is there any official image we can reference on our deployments? Any plan to add or the recommendation is, and will continue to ourselves build and push container images on our own repositories?

**Comment by mac-chaffee** [comment]
I see that OWASP has a dockerhub account already: https://hub.docker.com/u/owasp/#!

That might be something the coraza folks could gain access to

**Comment by jcmoraisjr** [comment]
Folks, we've these two unofficial images, which help the users from a component we develop to get started fast with Coraza:

```shell
docker.io/jcmoraisjr/coraza-spoa:experimental
quay.io/jcmoraisjr/coraza-spoa:experimental
```

The tags belong to a single image, same hash, and it adds #36 with all the suggestions already described in that PR.

We'd love to see it instead on a proper official docker repo. Thanks!

**Comment by jcchavezs** [comment]
Coraza spoa should publish it's own image in the coraza git repo (same as
coraza-proxy-wasm for example). Are you up to do a PR for that? Mainly the
GitHub action in coraza-proxy-wasm can be reused.

**Comment by mac-chaffee** [comment]
@jcmoraisjr I can submit the PR on Monday. But don't let me stop you if you were wanting to do this yourself sooner :)

**Comment by jcmoraisjr** [comment]
Thanks @mac-chaffee be my guest, I need to spend some cycles in the controller backlog.

**Comment by mac-chaffee** [comment]
@jcchavezs did you mean that coraza-spoa should have a github action in this repo? Or do you really mean there should be a github action in the main coraza repo?

**Comment by jcchavezs** [comment]
I meant in this repo. Same as coraza-proxy-wasm

**Comment by jcchavezs** [comment]
Closed by  #41
